src/objective.jl

```diff
@@ -25,8 +25,14 @@ end
 
 Return the best known bound on the optimal objective value after a call to
 `optimize!(model)`.
+
+For scalar-valued objectives, this function returns a `Float64`. For
+vector-valued objectives, it returns a `Vector{Float64}`.
+
+In the case of a vector-valued objective, this returns the ideal point, that
```

Comment:
What is an "ideal point" ? Is this a technical term from the multi-objective field ?

Reply:
Yes. See https://en.wikipedia.org/wiki/Multi-objective_optimization ("ideal objective vector").

It's also plotted in the tutorial:

<img width="716" alt="image" src="https://user-images.githubusercontent.com/8177701/218975378-4c1ed7a2-4c27-49d1-94b4-676f75413e18.png">

Reply:
Ok, maybe write it in italic to signify its the first occurence of this technical word and maybe add a href to the wikipedia page

Reply:
Added italics. I don't think the href is useful. We don't link to other technical terms, and the docstring includes a `that is, ...` comment in non-jargon.